docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+====
+
+. From *Fleet → Agent Policies*, select *Create agent policy* to create a policy for the {fleet-server}:
++
+** Set a *name* to the policy, for example `Fleet Server Policy Kubernetes`
+** Do *not* select the option *collect system logs and metrics*. This option would add the `system integration` to the policy, and considering the {fleet-server} will run as a Kubernetes Pod without any visibility to the Kubernetes node, there won't be any real system to monitor.
+** Select the **output** that the {fleet-server} needs to use to contact {es}. This should be the output that was verified in the previous step.
+** If desired, add an **inactive agent unenrollment timeout** to automatically unenroll and invalidate API Keys after being inactive. This is useful on Kubernetes environments as the {fleet-server} Pods are ephemeral, and after restarts new {agent}s will appear in Fleet UI.
+
+. Open the created policy, and from the *Integrations* tab select *Add integration*:
++
+** Search and select the Fleet Server integration
+** Select *Add Fleet Server* to add the integration to the policy.
++
+At this point you can configure the integration settings per <<fleet-server-scalability>>.
+** When done, select on *Save and continue*. Do not add any {agent} at this stage.
+
+. Open the configured policy, which now includes the Fleet Server integration, and select *Actions* → *Add Fleet Server*. In the next dialog:
++
+* Confirm that the *policy for Fleet Server* is properly selected.
+* *Choose a deployment mode for security*:
+** If you select *Quick start*, the {fleet-server} generates a `self-signed` TLS certificate, and subsequent agents should be enrolled using the `--insecure` flag.
+** If you select *Production*, you provide a certificate and key to the {fleet-server} during the deployment, and subsequent agents will need to trust the certificate's CA.
+* Add your *Fleet Server Host* information:
+** This will be the URL used by the clients ({agent}s) to connect to the {fleet-server}. In *Production* mode, the {fleet-server} certificate must include the hostname part of the URL as an `x509 SAN`.
+** On Kubernetes environments this could be the name of the `Kubernetes service` or reverse proxy that exposes the {fleet-server} Pods.
+** This setting is not really used by the {fleet-server} itself, so don't worry much about it at this stage because you will be able to change it later on under *Fleet → Settings*.
+* Select **generate service token** to create a token for the {fleet-server}.
+* From *Install Fleet Server to a centralized host → Linux*, take note of the values of the following settings that will be needed for the {fleet-server} installation:
+** Service token(under `--fleet-server-service-token` parameter).
+** Fleet policy ID (under `--fleet-server-policy` parameter).
+** {es} URL (under `--fleet-server-es` parameter).
```

Comment:
```suggestion
** Service token (below the `--fleet-server-service-token` parameter).
** Fleet policy ID (below the `--fleet-server-policy` parameter).
** {es} URL (below the `--fleet-server-es` parameter).
```
To me, `below` sounds a bit better than `under`, but I'm not sure why. I also added a missing space.

Reply:
what about `specified by` or `provided after`? :)

Reply:
I really like `specified by`.